abci/example/dummyapp/main.go

```diff
@@ -79,6 +79,11 @@ func main() {
 
 // DummyNode implements NodeProvider.
 func DummyNode(config *cfg.Config, provider ipfs.NodeProvider, logger log.Logger) (*node.Node, error) {
+	ipfsDir := os.ExpandEnv(filepath.Join("$HOME", cfg.DefaultTendermintDir, "ipfs"))
```

Comment:
Should we be using the IPFS portion of the config here? like, `cfg.IPFS.RepoPath`

Reply:
Probably `ipfs.DefaultConfig().RootDir`.

Reply:
That would certainly work for most situations and would be simpler. why not use the passed in config though?